app.js

```diff
@@ -18,35 +18,46 @@ window.addEventListener('load', async () => {
     // call findCountries function with no arguments to fetch all countries (Slice A);
     findCountries();
     // Slice B: call asynchronous getContinents fetch function and set to response variable
+    const gotContinents = await getContinents();
     // Slice B: set the continents state to the response.data
+    continents = gotContinents.data;
     // Slice B: call displayContinentOptions function;
+    displayContinentOptions();
 });
 
 async function findCountries(continent) {
     // Slice A: call the asynchronous fetch function to get the countries
     // Slice C: add continent argument to getCountries function call
     // console log the response object to see all of the nested information returned
+    const foundCountries = await getCountries(continent);
     // Slice A: set the countries state to the response.data
+    countries = foundCountries;
```

Comment:
The comment above is saying to set countries = foundCountries.data which helps shape the response data we get from Supabase. This seems to be working without it though. Just wanted you to know why it said that and what it does.